wgengine/tstun/tun_test.go

```diff
@@ -0,0 +1,354 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package tstun
```

Comment:
empty line before this line, otherwise the copyright comment becomes package doc in the AST for godoc/etc